runtime/src/bank.rs

```diff
@@ -9868,8 +9868,14 @@ pub(crate) mod tests {
                 dup_account.lamports -= lamports;
                 to_account.lamports += lamports;
             }
-            keyed_accounts[0].try_account_ref_mut()?.lamports -= lamports;
-            keyed_accounts[1].try_account_ref_mut()?.lamports += lamports;
+            keyed_accounts[0]
```

Comment:
test code replacement

Reply:
or, this could return a result. I didn't realize at first glance that it was already a result function.

Reply:
Yeah I'd just `?` here instead of unwrap